src/main/scala/firrtl/annotations/Annotation.scala

```diff
@@ -77,6 +77,48 @@ trait SingleTargetAnnotation[T <: Named] extends Annotation {
   }
 }
 
+/** Annotate Multi Target in a single Annotation,
+  * rather than [[SingleTargetAnnotation]] which duplicate the [[Annotation]] based on the [[RenameMap]],
+  * [[MultiTargetAnnotation]] keeps a [[RenameMap]] in the targets for better data structure
```

Comment:
Reword to: "[[MultiTargetAnnotation]] keeps the renamed targets grouped within a single annotation."

Reply:
Sorry for my redundant documentations.